content/02-learn/07-differences-with-other-clients.mdx

```diff
@@ -7,7 +7,7 @@ metaTitle: Differences with other clients
 Several attributes separate Mantis from other Ethereum classic clients:
 
   - Mantis is the only client **built from scratch**, specifically for the Ethereum Classic Network
-  - It is built in [**Scala**](https://www.scala-lang.org/), an object oriented and functional programming language
+  - It is built in [**Scala**](https://www.scala-lang.org/), programming language that allows build software using object oriented design on high level (DDD, hexagonal architecture) and use functional programming principles: immutability, referential transparency and lazy IO on low level. Thanks to advanced type systems, we eliminate many errors. Moreover it supports an actor model that naturally fits into the cryptocurrency world, as well as other approaches to concurrency like STM or green fibers. Scala programs running on JVM enjoy high & reliability enterprise grade reliability.
```

Comment:
I would just change some wording, but otherwise, great additions.

> It is built in [**Scala**](https://www.scala-lang.org/), programming language that allows build software using object oriented design on a high level (DDD, hexagonal architecture) and uses functional programming principles: immutability, referential transparency and lazy IO on a low level. Thanks to its advanced type systems, it eliminates many errors. Moreover it supports an actor model that naturally fits the cryptocurrency paradigms, as well as other approaches to concurrency like STM or green fibers. Scala programs running on the JVM have high, enterprise grade reliability.

Reply:
Thank you 🙇  I changed slightly the last phrase: `Scala programs running on the JVM have high performance and enterprise grade reliability`

Is it now correct?

Reply:
I actually added all potential improvements in the text I quoted.

Reply:
I've applied your improvements @faraggi thank you :)